go/enclave/rpc/rpc_utils.go

```diff
@@ -75,79 +66,39 @@ func GetViewingKeyAddressForTransaction(tx *common.L2Tx) (gethcommon.Address, er
 	return sender, nil
 }
 
-// ExtractCallParamTo extracts and parses the `to` field of an eth_call request.
-func ExtractCallParamTo(callParams []byte) (*gethcommon.Address, error) {
-	var paramsJSONMap []interface{}
-	err := json.Unmarshal(callParams, &paramsJSONMap)
+// ConvertToCallMsg converts the interface to a *ethereum.CallMsg
+func ConvertToCallMsg(callMsgInterface interface{}) (*ethereum.CallMsg, error) {
+	callMsgBytes, err := json.Marshal(callMsgInterface)
```

Comment:
Due to type mismatch ethereum.CallMsg can't be cast from interface{} :(.

Instead of maintaining functions that cast between interface and type, this marshalling creates the correct type from the interface